PhotoKingdomAPI/PhotoKingdomAPI/Controllers/Manager.cs

```diff
@@ -1857,6 +1857,77 @@ public AttractionBase AttractionAdd(AttractionAddForm newItem)
                 return mapper.Map<AttractionBase>(addedItem);
             }
         }
+
+        public AttractionWithDetails AttractionEditWin(AttractionWithWin item)
+        {
+            if (item == null)
+            {
+                return null;
+            }
+            else
+            {
+                var a = ds.Attractions
+                    .Include("City")
+                    .Include("QueuedUploads")
+                    .Include("AttractionPhotoWars")
+                    .Include("Owners")
+                    .SingleOrDefault(o => o.Id == item.AttractionId);
+
+                if (a != null && a.Owners.Count == 0 && a.AttractionPhotowars.Count == 0)
+                {
+                    var resident = ds.Residents.Find(item.OwnerId);
+
+                    if (resident != null)
+                    {
+                        var owner = ds.ResidentAttractionOwns.Add(new ResidentAttractionOwn
+                        {
+                            Title = "Lord of " + a.Name,
+                            StartOfOwn = DateTime.Now,
+                            ResidentId = resident.Id,
+                            AttractionId = a.Id
+                        });
+
+                        var winningPhoto = ds.Photos.Add(new Photo
+                        {
+                            PhotoFilePath = item.PhotoImagePath,
+                            Lat = item.PhotoLat,
+                            Lng = item.PhotoLng,
+                            ResidentId = resident.Id
+                        });
+
+                        var photowar = ds.AttractionPhotowars.Add(new AttractionPhotowar
+                        {
+                            AttractionId = a.Id,
+                            StartDate = DateTime.Now,
+                            EndDate = DateTime.Now
+                        });
+
+                        var photowarUpload = ds.AttractionPhotowarUploads.Add(new AttractionPhotowarUpload
+                        {
+                            PhotoId = winningPhoto.Id,
+                            AttractionPhotowarId = photowar.Id
+                        });
+                        ds.SaveChanges();
+

```

Comment:
Now that you have the photowarupload Id, pass it to method  CreateAttractionOwnForPhotoUpload(int photoUploadId). It will create the own for u.